docs/api_v2/api_v2_reference.apib

```diff
@@ -25,15 +26,38 @@ from services that can only send GET requests.
 The API token can be found on the
-<a href="https://my.apify.com/account#/integrations" target="_blank" rel="nofollow">Integrations</a>
-page in the Apify app.
+[Integrations](https://my.apify.com/account#/integrations) page in the Apify app.
 
-**IMPORTANT: Do not share the API token with untrusted parties, or use it directly from a client-side code,
+**IMPORTANT**: Do not share the API token with untrusted parties, or use it directly from a client-side code,
 unless you fully understand the consequences!**
 
 Note that some API endpoints such as [Get list of keys](#reference/key-value-stores/key-collection/get-list-of-keys)
 do not require any authentication token, because they contain a hard-to-guess identifier that effectively serves as an authentication key.
 
+## Basic usage
+
+To run an actor, send a POST request to the [Run actor](#reference/actors/run-collection/run-actor) endpoint using either the actor ID code (e.g. `vKg4IjxZbEYTYeW8T`) or its name (e.g. `janedoe~my-actor`):
+
+`https://api.apify.com/v2/acts/[actor_id]/runs`
+
+If the actor is not runnable anonymously, you will receive a 401 or 403
+[response code](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status).
+This means you need to append your [secret API token](https://my.apify.com/account#/integrations) as a query parameter `?token=[your_token]`.
+
+Optionally, you can include the query parameters described in the [Run actor](#reference/actors/run-collection/run-actor) section to customize your run.
+
+If you're using Node.js, the best way to run an actor is using the `Apify.call()` method from the [Apify SDK](https://sdk.apify.com/docs/api/apify#apifycallactid-input-options). It runs the actor using the account you are currently logged into (determined by the [secret API token](https://my.apify.com/account#/integrations)). The result is an [actor run object](https://sdk.apify.com/docs/typedefs/actor-run) and its output (if any). 
+
+A typical workflow is as follows:
+
+1. Run an actor or task using the [Run actor](#reference/actors/run-collection/run-actor) or [Run task](#reference/actor-tasks/run-collection/run-task) API endpoints.
+
+2. Monitor the actor run by periodically polling its progress using the [Get run](#reference/actors/run-object/get-run) API endpoint.
+
+3. Fetch the results from the [Get items](#reference/datasets/item-collection/get-items) API endpoint using the `defaultDatasetId`, which you receive in the Run request response. Additional data may be stored in a key-value store. You can fetch them from the [Get record](#reference/key-value-stores/record/get-record) API endpoint using the `defaultKeyValueStoreId` and the store's `key`.
+
+**Note**: Instead of periodic polling, you can also run your [actor](#reference/actors/run-actor-synchronously) or [task](#reference/actor-tasks/runs-collection/run-task-synchronously) synchronously. This will ensure that the request waits until the run finishes and returns its output.
```

Comment:
I'd mention here that these endpoints have timeout of 5 minutes during which the actor must finish otherwise an error is thrown.